src/test/java/tests/FilterEvalTests.java

```diff
@@ -455,6 +473,12 @@ public void author() {
         assertEquals(false, matches("author:alice", issue));
         assertEquals(true, matches("author:o", issue));
         assertEquals(false, matches("author:lic", issue));
+
+        assertEquals(true, matches("au:BOB", issue));
+        assertEquals(true, matches("au:bob", issue));
+        assertEquals(false, matches("au:alice", issue));
+        assertEquals(true, matches("au:o", issue));
+        assertEquals(false, matches("au:lic", issue));
```

Comment:
Instead of duplicating the lines, can we abstract it to a method and call twice?